src/vs/workbench/services/suggest/browser/simpleCompletionModel.ts

```diff
@@ -207,6 +207,7 @@ export class SimpleCompletionModel {
 				}
 				// Then by file extension length ascending
 				score = a.fileExtLow.length - b.fileExtLow.length;
+				score = a.completion.label.localeCompare(b.completion.label);
```

Comment:
Ah that explains the alphabetical ordering. This change clobbers the extension length part so make sure that doesn't break.